src/index.template.html

```diff
@@ -701,6 +701,17 @@
                                 <label class="btn btn-xwide" for="srR">Stealth Rock</label>
                             </div></td>
                         </tr>
+                        <tr class="gen-specific g4 g5 g6 g7 g8">
```

Comment:
I don’t think steel spikes was around in gens 4, 5, 6, and 7 O.o

Reply:
Done